spec/Powertrain/CombustionEngine.vspec

```diff
@@ -230,6 +230,42 @@ Torque:
   description: Current engine torque.
 
 
+#
+# Diesel Exhaust Fluid
+#
+
+DieselExhaustFluid:
+  type: branch
+  description: Signals related to Diesel Exhaust Fluid (DEF).
+               DEF is called AUS32 in ISO 22241.
+  comment: In retail and marketing other names are typically used for the fluid.
+
+DieselExhaustFluid.Capacity:
+  datatype: float
+  type: attribute
+  unit: l
+  description: Capacity in liters of the Diesel Exhaust Fluid Tank.
+
+DieselExhaustFluid.Level:
+  datatype: uint8
+  type: sensor
+  unit: percent
+  min: 0
+  max: 100
+  description: Level of the Diesel Exhaust Fluid tank as percent of capacity. 0 = empty. 100 = full.
+
+DieselExhaustFluid.Range:
+  datatype: uint32
+  type: sensor
+  unit: m
+  description: Remaining range in meters of the Diesel Exhaust Fluid present in the vehicle.
+
+DieselExhaustFluid.IsLevelLow:
+  datatype: boolean
+  type: sensor
+  description: Indicates if the Diesel Exhaust Fluid level is low.
```

Comment:
Nitpicking mode: Any specific reason why this description is multi-line, while the others are not?

Reply:
Not really, Three sentences so would be quite long if put on the same line. But we have not really defined any guidelines on recommended line length, or? Might possibly be an idea to add?

Reply:
Let's not overdo explicitly spelling out best practices :) I see your argument, but the description would still fit into two reasonable sized lines... 🙈 
Anyway, approving it now, because the signals are what we are missing 👍